fluidBuild.config.cjs

```diff
@@ -77,29 +88,30 @@ module.exports = {
 		// Generic build:test script should be replaced by :esm or :cjs specific versions.
 		// "tsc" would be nice to eliminate from here, but plenty of packages still focus
 		// on CommonJS.
-		"build:test": ["typetests:gen", "tsc", "api-extractor:commonjs", "api-extractor:esnext"],
-		"build:test:cjs": ["typetests:gen", "tsc", "api-extractor:commonjs"],
-		"build:test:esm": ["typetests:gen", "build:esnext", "api-extractor:esnext"],
+		"build:test": ["typetests:gen", "tsc"],
+		"build:test:cjs": ["typetests:gen", "tsc", "build:entrypoints:commonjs"],
+		"build:test:esm": ["typetests:gen", "build:esnext", "build:entrypoints:esnext"],
 		"api": {
-			dependsOn: ["api-extractor:commonjs", "api-extractor:esnext"],
+			dependsOn: ["build:entrypoints:esnext:cjs"],
 			// dependsOn: ["api-extractor:commonjs", "api-extractor:esnext"],
 			script: false,
 		},
-		"api-extractor:commonjs": ["tsc"],
-		"api-extractor:esnext": {
-			dependsOn: ["build:esnext"],
+		"build:entrypoints:commonjs": ["tsc", "build:entrypoints"],
```

Comment:
Based on our naming style, I would expect build:entrypoints:commonjs to be a part of build:rntrypoints, not depend on it. Can you describe the new build order with these changes?

Reply:
The build order would be:
1. build:esnext
2. generate sourceEntrypoints - build:entrypoints
3. compile source entrypoints - build:entrypoints:esnext, build:entrypoints:cjs 
4. generate node10

Reply:
Ouch, so we have to build twice.

Reply:
Why does step 2 depend on step 1? Does the project have to be built in order to generate these files? If the input is a TS file, then a build should not be needed.

Why do the entrypoints have to be built separately from the main project? Can the files not be generated then just built as part of the regular project? Seems like either way you have to update the exports field paths manually since they're changing.

Reply:
The input is whole of the current package production (less the entrypoints). If any of the project changes the entrypoints will (may) need to be regenerated just like they are today. The source generation task is best as a TscDependentTask.